types/tui-code-snippet.d.ts

```diff
@@ -16,13 +16,29 @@ declare module 'tui-code-snippet/type/isExisty' {
   export default function isExisty(obj: any): obj is boolean;
 }
 
+declare module 'tui-code-snippet/type/isFunction' {
+  export default function isFunction(obj: any): obj is Function;
+}
+
+declare module 'tui-code-snippet/type/isNumber' {
+  export default function isNumber(obj: any): obj is number;
+}
+
+declare module 'tui-code-snippet/type/isObject' {
+  export default function isObject(obj: any): obj is object;
+}
+
+declare module 'tui-code-snippet/type/isArray' {
+  export default function isArray(obj: any): obj is Array<any>;
+}
+
 // object
 declare module 'tui-code-snippet/object/extend' {
   export default function extend<T>(obj: T, ...objects: T[]): T;
 }
 
 declare module 'tui-code-snippet/object/pick' {
-  export default function pick(obj: any, ...paths: string[]): any;
+  export default function pick(obj: any, ...paths: string[] | number[]): any;
```

Comment:
Will `pick`'s return type be a string array or a numeric array?

Reply:
The return type can be any value. This is the [doc](http://nhn.github.io/tui.code-snippet/latest/object#pick).